sqldelight-compiler/integration-tests/src/test/kotlin/com/example/GroupQueries.kt

```diff
@@ -12,4 +14,22 @@ public class GroupQueries(
       "selectAll", "SELECT `index` FROM `group`") { cursor ->
     cursor.getLong(0)!!
   }
+
+  public fun <T : Any> selectFromTable2(mapper: (something: String?, nice: String?) -> T): Query<T>
+      = Query(-620576550, arrayOf("myftstable2"), driver, "Group.sq", "selectFromTable2", """
+  |SELECT *
+  |FROM myftstable2
+  """.trimMargin()) { cursor ->
+    mapper(
+      cursor.getString(0),
+      cursor.getString(1)
+    )
+  }
+
+  public fun selectFromTable2(): Query<Myftstable2> = selectFromTable2 { something, nice ->
```

Comment:
I struggle to understand this type. This is the type of the virtual table. According to the PR, this would not be generated. And with a clean build, this type isn't generated. Is this expected? If so, how does the CI pass?

Reply:
We should rerun the integration test twice, the first run generates (in-) valid code, the second run validates the code.